server/shadow/shadow_client.c

```diff
@@ -817,85 +817,6 @@ int shadow_client_surface_update(rdpShadowClient* client, REGION16* region)
 	return 1;
 }
 
-int shadow_client_convert_alpha_pointer_data(BYTE* pixels, BOOL premultiplied,
```

Comment:
I was wondering if it would't have made sense to keep as much in the generic part as possible that different concrete implementations can re-use it?